agent/bench-scripts/postprocess/linpack-postprocess

```diff
@@ -29,105 +30,100 @@ my %throughput;
-	closedir($clients_dh);
+        @client_data_files = grep(/^linpack\..+$/, (sort readdir($clients_dh)));
+        closedir($clients_dh);
     } else {
-	printf STDERR "ERROR: Could not open client %s results directory!\n", $client_hostname_dir;
-	exit 2;
+        printf STDERR "ERROR: Could not open client %s results directory!\n", $client_hostname_dir;
+        exit 2;
     }
 
     foreach my $client_data_file (@client_data_files) {
-	if ($client_data_file =~ /^linpack\.(out|meta)$/) {
-	    if (open(CLIENT_OUTPUT, "$dir/clients/$client_hostname_dir/$client_data_file")) {
-		while(<CLIENT_OUTPUT>) {
-		    if ($client_data_file =~ /meta$/) {
-			if ($_ =~ /^(binary|problem_sizes|leading_dimensions|alignment_values|numactl_cmd|use_omp|kmp_affinity|threads)/) {
-			    chomp($_);
-			    my @fields = split(/=/, $_, 2);
-			    $client_data{$client_hostname_dir}{$fields[0]} = $fields[1];
-			}
-		    } elsif ($client_data_file =~ /out$/) {
-			if ($_ =~ /^[0-9]+/) {
-			    chomp($_);
-			    my @fields = split(/\s+/, $_);
-			    if (@fields == 5) {
-				$client_data{$client_hostname_dir}{'bw'} = $fields[3];
-			    }
-			} elsif ($_ =~ /^Performance Summary/) {
-			    $_ =~ m/\((.+)\)$/;
-			    $client_data{$client_hostname_dir}{'bw_units'} = $1;
-			}
-		    }
-		}
-	    } else {
-		printf STDERR "ERROR: Could not open client %s result file %s!\n", $client_hostname_dir, $client_data_file;
-		exit 3;
-	    }
-	}
+        if ($client_data_file =~ /^linpack\.(out|meta)$/) {
+            if (open(CLIENT_OUTPUT, "$dir/clients/$client_hostname_dir/$client_data_file")) {
+                while (<CLIENT_OUTPUT>) {
+                    if ($client_data_file =~ /meta$/) {
+                        if ($_ =~ /^(binary|problem_sizes|leading_dimensions|alignment_values|numactl_cmd|use_omp|kmp_affinity|threads)/) {
+                            chomp($_);
+                            my @fields = split(/=/, $_, 2);
+                            $client_data{$client_hostname_dir}{$fields[0]} = $fields[1];
+                        }
+                    } elsif ($client_data_file =~ /out$/) {
+                        if ($_ =~ /^[0-9]+/) {
+                            chomp($_);
+                            my @fields = split(/\s+/, $_);
+                            if (@fields == 5) {
+                                $client_data{$client_hostname_dir}{'bw'} = $fields[3];
+                            }
+                        } elsif ($_ =~ /^Performance Summary/) {
+                            $_ =~ m/\((.+)\)$/;
+                            $client_data{$client_hostname_dir}{'bw_units'} = $1;
+                        }
+                    }
+                }
+            } else {
+                printf STDERR "ERROR: Could not open client %s result file %s!\n", $client_hostname_dir, $client_data_file;
+                exit 3;
+            }
+        }
     }
 }
 
 print Dumper \%client_data;
```

Comment:
Is this debugging output? Should it be conditionally executed? Or commented out?

Reply:
I'm afraid that I have no idea; and, I don't see any obvious tests for this script, so I can't really comment.  But, I will say, since I didn't add this line, I'm not really inclined to remove it....